.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# HTCondor Pull Request Checklist for internal reviewers
+
+- [] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
+- [] Verify that the related Jira ticket exists and has a target version number and that it is correct.
+- [] Verify that the Jira ticket is in review status and is assigned to the reviewer.
+- [] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
+- [] Verify that the branch destination of the PR matches the target version of the ticket
+- [] Check for correctness of change
+- [] Check for documentation, if needed
+- [] Check for version history, if needed
+- [] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) ) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
+- [] Check that commit message references the Jira ticket (HTCONDOR-xxx)
```

Comment:
Apparently a space is needed in the square brackets to turn these into checkboxes:

```suggestion
- [ ] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [ ] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [ ] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [ ] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for documentation, if needed
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) ) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [ ] Check that commit message references the Jira ticket (HTCONDOR-xxx)
```